### DISCUSSION TITLE
unautorised payment deduction

### DISCUSSION THREAD

**Discussion by Piyush12111992** [community/community#56408] [New to GitHub] [answered]
I didnt authorize the payment but they deducted 100$ from my account. How do i get my money back

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Piyush12111992 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by hengsok9292** [comment]
> I didnt authorize the payment but they deducted 100$ from my account. How do i get my money back

**Comment by regissoren7** [comment]
have canceled my Copilot subscription, but you still sent me an invoice — why?![image](https://github.com/user-attachments/assets/01ce6d64-200b-4d5b-a4ad-f82e9da73273)
